homeassistant/components/samsungtv/__init__.py

```diff
@@ -249,7 +249,7 @@ async def _async_create_bridge_with_updated_data(
                 updated_data[CONF_MODEL] = model
 
     if model_requires_encryption(model) and method != METHOD_ENCRYPTED_WEBSOCKET:
-        LOGGER.warning(
+        LOGGER.debug(
```

Comment:
This whole code block `_async_create_bridge_with_updated_data` needs to be reviewed now that it is all going through the config flow.
Marking this for beta to avoid unnecessary noise.

Reply:
For reference, from @MartinHjelmare :
> I think this message comes at the wrong time. It should be a notice in the config flow if this case happens, since that's when the connection method is detected. If the user would be able to do anything about it, we could create a repair issue, but that doesn't seem to be the case.

> So I'd remove the log here, and add a step in the config flow just to show this warning if that case happens. Let the user then decide to proceed or abort the flow.